packages/interactivity/src/utils.ts

```diff
@@ -55,6 +69,14 @@ const afterNextFrame = ( callback: () => void ) => {
 	} );
 };
 
+// TODO: consider initializing this only if required.
+const taskQueue: ( () => any )[] = [];
+const taskChannel = new MessageChannel();
+// Drain one task per message
+taskChannel.port1.onmessage = () => {
+	taskQueue.shift()?.();
+};
```

Comment:
I'm reusing these for a `scheduler` fallback. Not entirely confident this is the most suitable approach, though. 🤔

Using `MessageChannel` could be considered more reliable than `setTimeout()` to schedule tasks (the [`scheduler` polyfill](https://github.com/GoogleChromeLabs/scheduler-polyfill) from GoogleChromeLabs uses it when available), I feel that none of the [delay issues](https://developer.mozilla.org/en-US/docs/Web/API/Window/setTimeout#reasons_for_delays_longer_than_specified) `setTimeout()` has would apply here, at least for the initialization case.